azure-stack/hci/manage/arc-extension-management.md

```diff
@@ -145,6 +209,21 @@ If desired, you can uninstall some extensions from your Azure Stack HCI clusters
 
     :::image type="content" source="media/arc-extension-management/arc-extension-uninstall-extension-2.png" alt-text="Screenshot of the notification to uninstall an extension in the Azure portal." lightbox="media/arc-extension-management/arc-extension-uninstall-extension-2.png":::
 
+# [Azure CLI](#tab/azurecli)
+To remove a specific extension like `AzureMonitorWindowsAgent` run the following command:
+    ```azurecli
+    extensionName="AzureMonitorWindowsAgent" # Replace with the extension  name
+    resourceGroup="hcicluster-rg" # Replace with your resource group name
+    clusterName=""HCICluster" # Replace with your cluster name
+    extensionName="AzureMonitorWindowsAgent" # Replace with the extension  name
+
```

Comment:
This code block is not showing up correctly, remove the empty space before az stack-hci extension delete \

Reply:
Removed